sleap_nn/config/model_config.py

```
@@ -0,0 +1,276 @@
+    init_weight: str = "default"
+    pre_trained_weights: str = None
+    backbone_config: BackboneConfig = attrs.field(factory=BackboneConfig)
+    head_configs: HeadConfig = attrs.field(factory=HeadConfig)
+
+    # post-initialization
+    def __attrs_post_init__(self):
+        self.validate_pre_trained_weights()
+
+    # validate the pre-trained weights
+    def validate_pre_trained_weights(self):
+        convnext_weights = [
```

Ruff (0.8.2)

Local variable `convnext_weights` is assigned to but never used

Remove assignment to unused variable `convnext_weights`

(F841)